JS/Lesson25/sinan/app/script.js

```diff
@@ -14,6 +14,8 @@ const button = document.getElementById('fetch-button');
 
 function fetchImages() {
   // TODO: Clear screen before loading new images
+  const container = document.getElementById("image-container");
```

Comment:
You don't need to define the container variable again, it was already defined on the line 11 so you can just reuse it here :)